src/entities/Promotion.ts

```diff
@@ -21,6 +21,12 @@ export class Promotion {
   @Column()
   description: string;
 
+  @Column('decimal', { precision: 6, scale: 2 })
+  price: number;
+
+  @Column('decimal', { precision: 6, scale: 2 })
+  promotion: number;
```

Comment:
What's a promotion here ? What is it equivalent to ?

Reply:
Promotion is the reductions that is applied on the product, have you a more relevant name for this column ?